mars/services/cluster/uploader.py

```diff
@@ -154,15 +155,19 @@ async def upload_node_info(self, status: NodeStatus = None):
             raise
 
     def get_bands(self) -> Dict[BandType, int]:
-        band_slots = dict()
+        band_resource = dict()
         for resource_type, info in self._info.resource.items():
             if resource_type.startswith("numa"):
                 # cpu
-                band_slots[(self.address, resource_type)] = info["cpu_total"]
+                band_resource[(self.address, resource_type)] = Resource(
+                    num_cpus=info["cpu_total"], num_mem_bytes=info["memory_total"]
```

Comment:
```suggestion
                    num_cpus=info["cpu_total"], mem_bytes=info["memory_total"]
```

Reply:
`num_mem_bytes` here is  a `Resource` parameter in `mars/_resource.pyx` which is merged in #2732. How about rename it in a later pr if we are sure to fix it?

Reply:
Just fix it in this PR.

Reply:
OK.